compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5795,6 +5795,11 @@ TR::Register *OMR::Power::TreeEvaluator::sbyteswapEvaluator(TR::Node *node, TR::
          {
          generateTrg1Src1Instruction(cg, TR::InstOpCode::brh, node, tgtRegister, srcRegister);
          }
```

Comment:
the only potential problem i can see is the mask constants which might not be accepted by the compiler (e.g.  32bit mode build).

Reply:
I tried to stay consistent with the other instructions that uses 5-bytes mask constant for some reason... I'll add a new commit that changes all of them to 32-bit mask constant.

Reply:
no, you might misunderstand it.  i had impressions that mask constants are offered in CONSTANT64(0x....).

Reply:
Added in 500c36c